GVFS/GVFS.UnitTests.Windows/Windows/CommandLine/SparseVerbTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using GVFS.CommandLine;
+using GVFS.Tests.Should;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace GVFS.UnitTests.Windows.Windows.CommandLine
+{
+    [TestFixture]
+    public class SparseVerbTests
+    {
+        // Shorter name for readability
+        private static char dirSep = Path.DirectorySeparatorChar;

```

Comment:
Since this is only Windows why can't we just use a `\`?

Reply:
We could do that, and I will make the change.

I went back and forth on the best way to set these up.  The trouble is these tests aren't really *Windows* specific, but they depend on the `GVFS.Windows` project (which is Windows specific).

I will update these to use `\`, and if we decide to make the test cross-plat at some point we can move the code into a .NET Standard assembly or update `GVFS.Windows` to be .NET Core (assuming ProjFS is by that point).